plutus/budget_manager/mysql.py

```diff
@@ -0,0 +1,162 @@
+from constants import APP
```

Comment:
I think this file should be renamed from `mysql.py` to `db.py`, `backend.py`, `state.py` to make it more generic to operations against your state storage.

Reply:
is it fair to leave this as is? I don't anticipate using any other form of db, and adding another would need more refactorization anyways (cli + imports + sql create table + query changes). I could cross that bridge in the future if need be

Reply:
That is fine.